players/installing-java/windows.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Installing Java on Windows
+description: A step by step guide on how to install Java on Windows.
+---
+
+# Installing Java on Windows
+
+Java is required to run Minecraft and the majority of third party launchers. This guide will show you how to install Java on Windows.
```

Comment:
I think these Java pages will be better of in the developer section tbh. The vanilla launcher and basically all 3rd party launchers (other than 1 major one) handle the downloading of Java for you.

The fabric installer (on windows) also goes looking for Java included with Minecraft, meaning that most of the players seeking help wont need to install Java.

I think its going to be out of scope for us to provide details on using 3rd party launchers, that would be better suited to the main wiki.

Reply:
Will move them once the develop section starts popping up.